flink-staging/flink-hbase/pom.xml

```diff
@@ -112,6 +112,12 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
```

Comment:
But putting it into test scope is not a proper solution to solve the issue. 
IMO, it should be either put in the regular scope such that it can be used at runtime or we put it into a comment and add a line explaining why we did that.